### PR TITLE
🧹 add integration MRN label for node scanning

### PR DESCRIPTION
**Pull request by joelddiaz**
When the MondooAuditConfig has consoleIntegration enabled, ensure
that we set up a Mondoo inventory that includes the inventory MRN
label like we already do for cluster scanning and the webhook.

closes: #387 

Signed-off-by: Joel Diaz <joel@mondoo.com>

### CODE REVIEW

**Comment by joelddiaz** [comment]
cc @vjeffrey

File: controllers/nodes/resources.go [hunk 1]
```diff
@@ -173,9 +172,46 @@ func OldConfigMapName(prefix string) string {
 	return fmt.Sprintf("%s%s", prefix, OldInventoryConfigMapSuffix)
 }
 
-func Inventory(node corev1.Node, m v1alpha2.MondooAuditConfig) string {
-	inventory := string(inventoryYaml)
-	return strings.ReplaceAll(inventory, InventoryNodeNamePlaceholder, node.Name)
+func Inventory(node corev1.Node, integrationMRN string, m v1alpha2.MondooAuditConfig) (string, error) {
+	inv := inventory.MondooInventory{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "mondoo-k8s-inventory",
```
Comment (imilchev) [inline controllers/nodes/resources.go:178]: can we change the name to `mondoo-node-inventory` or something along these lines?

**Review by imilchev — COMMENTED** [review]
Apart from that nit it looks good! Let's ship :shipit:

**Review by imilchev — APPROVED** [review]

**Review by vjeffrey — APPROVED** [review]

File: controllers/nodes/resources.go [hunk 2]
```diff
@@ -89,6 +89,10 @@ func CronJob(image string, node corev1.Node, m v1alpha2.MondooAuditConfig) *batc
 											Value: "on",
 										},
 									},
+									SecurityContext: &corev1.SecurityContext{
+										//RunAsUser:    pointer.Int64(9999),
```
Comment (imilchev) [inline controllers/nodes/resources.go:93]: why the comment?

File: controllers/nodes/resources.go [hunk 3]
```diff
@@ -89,6 +89,10 @@ func CronJob(image string, node corev1.Node, m v1alpha2.MondooAuditConfig) *batc
...
+										RunAsNonRoot: pointer.Bool(true),
```
Comment (imilchev) [inline controllers/nodes/resources.go:94]: does that even work? The user within the mondoo client container is actually `root`. See [here](https://github.com/mondoohq/client/blob/master/Dockerfile#L27-L33)

File: controllers/nodes/resources.go (hunk 2 above)
Comment (joelddiaz) [reply controllers/nodes/resources.go:93]: That was some other thing I was working on...I'll remove

File: controllers/nodes/resources.go (hunk 3 above)
Comment (joelddiaz) [reply controllers/nodes/resources.go:94]: Same thing...working on a separate thing, and it snuck into this PR

**Pull request merged**